Sources/TABResourceLoader/Services/NetworkDataResourceService.swift

```diff
@@ -38,17 +38,22 @@ open class NetworkDataResourceService<NetworkDataResource: NetworkResourceType &
   }
   
   /**
-   Designated initilizer for NetworkDataResourceService
+   Designated initializer for NetworkDataResourceService
    
    - returns: An new instance
    */
   public required init() {
```

Comment:
do we still need this initialiser if they are both `public`? maybe this could go and we could add a default parameter to the initialiser that is being made `public`